website/content/docs/discovery/dns.mdx

```diff
@@ -444,34 +550,53 @@ If you need more complex behavior, please use the
 
 ### Service Virtual IP Lookups
 
-To find the unique virtual IP allocated for a service:
+Service virtual IP lookups are only applicable when performed from a downstream
+service in the mesh with [transparent proxy](/docs/connect/transparent-proxy) enabled.
+Such a downstream service's application code can directly reference its upstream with
+a service virtual IP lookup, rather than referencing a `localhost:port` combination
+that maps to an [explicit upstream](/consul/docs/k8s/annotations-and-labels#consul-hashicorp-com-connect-service-upstreams).
+To use this Consul DNS lookup format in Consul on Kubernetes,
+set [`dns.enableRedirection`](/consul/docs/k8s/helm#v-dns-enableredirection) to `true`.
+
+A service virtual IP lookup returns a unique virtual IP allocated for a
+[Connect-capable](/docs/connect) service. This virtual IP is also included in the
+service's [Tagged Addresses](/docs/discovery/services#tagged-addresses)
+under the `consul-virtual` tag.
+
+Use the following query format to lookup a service, optionally in a cluster peer:
 
 ```text
 <service>.virtual[.<peer>].<domain>
 ```
 
-This will return the unique virtual IP for any [Connect-capable](/docs/connect)
-service. Each Connect service has a virtual IP assigned to it by Consul - this is used
-by sidecar proxies for the [Transparent Proxy](/docs/connect/transparent-proxy) feature.
-The peer name is an optional part of the FQDN, and it is used to query for the virtual IP
-of a service imported from that peer.
+To lookup services outside the context of a mesh service's application code
+with transparent proxy enabled, refer to non-virtual
+[service lookups](#service-lookups) instead.
 
-The virtual IP is also added to the service's [Tagged Addresses](/docs/discovery/services#tagged-addresses)
-under the `consul-virtual` tag.
- 
 #### Service Virtual IP Lookups for Consul Enterprise <EnterpriseAlert inline />
 
 By default, a service virtual IP lookup uses the `default` namespace
 within the partition and datacenter of the Consul agent that received the DNS query.
 
-To lookup services imported from a cluster peered partition or open-source datacenter,
-specify the namespace and peer name in the lookup:
-```text
-<service>.virtual[.<namespace>].<peer>.<domain>
-```
+Use the following query formats to perform a service virtual IP lookup
+on namespaced or partitioned services:
+
+- Use the [canonical format](#canonical-format) to optionally specify
+  namespace, peer, and/or partition:
+
+  ```text
+  <service>.virtual[.<namespace>.ns][.<peer>.peer][.<partition>.ap].<domain>
+  ```
+
+- Specify cluster peer and optional namespace in the lookup:
```

Comment:
```suggestion
- Use the [canonical format](#canonical-format) to specify
  namespace, peer, and/or partition:

  ```text
  <service>.virtual[.<namespace>.ns][.<peer>.peer][.<partition>.ap].<domain>
  ```

- Specify cluster peer and namespace in the lookup:
```
Do we need to keep prefacing as these parts as optional?